includes/gateways/stripe/includes/admin/class-give-stripe-admin-settings.php

```diff
@@ -676,105 +687,195 @@ public function register_advanced_settings( $settings ) {
-							<span><?php echo __( 'Connected', 'give' ); ?></span>
-						</span>
-						<p class="give-field-description">
-							<span class="dashicons dashicons-yes" style="color:#25802d;"></span>
-							<?php
-							esc_attr_e( 'Stripe is connected.', 'give' );
-							$disconnect_confirmation_message = sprintf(
-								/* translators: %s Stripe User ID */
-								__( 'Are you sure you want to disconnect GiveWP from Stripe? If disconnected, this website and any others sharing the same Stripe account (%s) that are connected to GiveWP will need to reconnect in order to process payments.', 'give' ),
-								$stripe_user_id
-							);
-							?>
-							<a href="<?php give_stripe_disconnect_url(); ?>" class="give-stripe-disconnect">
-								<?php esc_attr_e( '[Disconnect]', 'give' ); ?>
-							</a>
-						</p>
-					<?php else : ?>
-						<?php echo give_stripe_connect_button(); ?>
-						<p class="give-field-description">
-							<span class="dashicons dashicons-no"
-								style="color:red;"></span><?php esc_html_e( 'Stripe is NOT connected.', 'give' ); ?>
-						</p>
-						<?php if ( isset( $_GET['error_code'] ) && isset( $_GET['error_message'] ) ) : ?>
-							<p class="stripe-connect-error">
-								<strong><?php echo give_clean( $_GET['error_code'] ); ?>:</strong> <?php echo give_clean( $_GET['error_message'] ); ?>
-							</p>
-						<?php endif; ?>
-					<?php endif; ?>
-					<?php
-					if ( ! defined( 'GIVE_STRIPE_VERSION' ) ) {
-						?>
-						<p class="give-field-description">
+						</div>
+						<div class="give-stripe-account-manager-list">
 							<?php
-							echo sprintf(
-								__( 'The free Stripe payment gateway includes an additional 2%% fee for processing one-time donations. This fee is removed by activating the premium <a href="%1$s" target="_blank">Stripe add-on</a> and never applies to subscription donations made through the <a href="%2$s" target="_blank">Recurring Donations add-on</a>. <a href="%3$s" target="_blank">Learn More ></a>', 'give' ),
-								esc_url( 'https://givewp.com/addons/stripe-gateway/' ),
-								esc_url( 'https://givewp.com/addons/recurring-donations/' ),
-								esc_url( 'http://docs.givewp.com/addon-stripe' )
-							);
-							?>
-						</p>
-						<?php
-					}
-					?>
+							if ( is_array( $stripe_accounts ) && count( $stripe_accounts ) > 0 ) {
+								foreach ( $stripe_accounts as $name => $details ) {
+									$stripe_account_id  = ! empty( $details['give_stripe_user_id'] ) ? $details['give_stripe_user_id'] : '';
+									$disconnect_message = ( 'connect' === $details['type'] ) ?
+										sprintf(
+											__( 'Are you sure you want to disconnect GiveWP from Stripe? If disconnected, this website and any others sharing the same Stripe account (%1$s) that are connected to GiveWP will need to reconnect in order to process payments.', 'give' ),
+											$stripe_account_id
+										) :
+										__( 'Are you sure you want to disconnect GiveWP from Stripe?', 'give' );
```

Comment:
Because translated strings are user-defined, the translated string should be escaped.